packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -112,6 +124,7 @@ export default function EditorStyles( { styles } ) {
 				} }
 				dangerouslySetInnerHTML={ { __html: transformedSvgs } }
 			/>
+			{ children }
```

Comment:
Do we still need to render `children`? The component is no longer a `Provider` wrapper - 715d47f62220dbf2854e2814d10fdb40912425df.